coalib/bears/GlobalBear.py

```diff
@@ -4,22 +4,28 @@
 
 class GlobalBear(Bear):
     """
-    A GlobalBear is able to analyze semantic facts across several file.
+    A GlobalBear analyzes semantic facts across several files.
 
     The results of a GlobalBear will be presented grouped by the origin Bear.
-    Therefore Results spanning above multiple files are allowed and will be
-    handled right.
+    Therefore Results spanning across multiple files are allowed and will be
+    handled correctly.
 
-    If you only look at one file at once anyway a LocalBear is better for your
-    needs. (And better for performance and usability for both user and
-    developer.)
+    If you are inspecting a single file at a time, you should consider
+    using a LocalBear.
     """
 
     def __init__(self,
-                 file_dict,  # filename : file contents
+                 file_dict,
                  section,
                  message_queue,
                  timeout=0):
+        """
+        Constructs a new GlobalBear.
+
+        :param file_dict: The dictionary of {filename: file contents}.
+
```

Comment:
> I've not heard of "metaclass conflict" before, so yeah, let's wait for third opinion :)

By that I mean the `LocalLinterMeta` and `GlobalLinterMeta` which respectively have `LocalLinterBase` and `GlobalLinterBase` classes as derivatives. These latter classes each have `create_arguments` staticmethods that have very similar documentation. The phrase "metaclass conflict" is in the docstring of both `*Meta` classes and that is what I was referring too.

So, bottomline, I understand the copy-pastedness of those docs as consequence of that "metaclass conflict". No idea if I am right/wrong though.

And the bottomline for current conflict: is it necessary to copy-paste docs from `Bear` into `GlobalBear`? Or does the doc reference from GlobalBear into Bear work?